src/components/card/AttachmentList.vue

```diff
@@ -234,6 +234,7 @@ export default {
 						shareWith: '' + this.cardId,
 					}).then(() => {
 						this.$store.dispatch('fetchAttachments', this.cardId)
+						this.$store.commit('cardIncreaseAttachmentCount', this.cardId)
```

Comment:
Maybe updating the value direct into `fetchAttachments`  action will be more consistent.